e2e/cypress/e2e/local-ui-tests.cy.js

```diff
@@ -0,0 +1,28 @@
+import { accountDetails, pageHeadings, URL } from '../support/constants.js';
+
+describe('Local environment test', () => {
+    it('Opens VA homepage in the local environment', () => {
+      cy.visit(URL.localURL)

```

Comment:
This test doesn't assert anything. Cypress can be really slow setting up the context around each test - given the overhead, normally I would expect longer tests that do more things. So here I would expect all 5 tests to be combined in to one. The idea is to think of them more like UAT scripted tests, where a user is following a whole workflow. Your longer tests in Vector-Atlas-tests.cy are about right.

Reply:
I've merged all of the tests into one (for local and http). The reason I split them so it was easier to see exactly what was being tested.

Reply:
Yes, and that's exactly what I'd do for unit tests and if Cypress didn't have so much overhead. It's more of a performance change because of the drawback of long set up times.